styles/desktop.css

```diff
@@ -1,203 +1,230 @@
-  line-height: 78px;
-  padding-bottom: 24px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  padding-top: 10px;
+  max-width: 450px;
+}
+
+.top-page-social-media {
   order: 0;
-  letter-spacing: 0.7px;
 }
-.empty-divider1 {
-  padding-top: 20px;
+
+ul#top-page-social-media-list {
+  flex-direction: column;
+  padding-top: 55px;
 }
-.support-text1 {
-  color: white;
-  font-family: 'Inter', sans-serif;
-  font-weight: 500;
-  font-size: 32px;
-  line-height: 44px;
-  padding-bottom: 24px;
-  order: 1;
- 
+
+ul#top-page-social-media-list li {
+  display: inline;
+  margin: 10px;
 }
-.description1 {
-  color: #F4F5F7;
-  font-family: 'Inter', sans-serif;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 32px;
-  order: 2;
+
+/* Top Page Ends above */
+
+.my-recent-work {
+  display: grid;
+  height: 100vh;
+  background-color: #e5e5e5;
 }
+
 .my-recent-work-top {
   display: flex;
   flex-direction: row;
-  justify-content:flex-start;
-  padding-left: 107px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  padding-right: 24px;
+  padding-left: 36px;
 }
-.my-recent-work {
-  color: #091E42;
-  order: 0;
-  min-width: 325px;
+
+.my-recent-work-title {
+  width: 580px;
+  padding-top: 8px;
 }
-.my-recent-work-border-mobile {
-  display: none;
+
+.my-recent-work-horizontal-line {
+  width: 100%;
 }
-.my-recent-work-border-desktop {
-  display: block;
-  padding-top: 138px;
-  align-self: left;
+
+.mrw-hz-line {
+  margin-top: 40px;
+  border: none;
+  height: 1px;
+  background-color: black;
+  width: auto;
+  margin-left: 20px;
+  align-items: center;
 }
-.desktop-my-recent-work-border {
-  max-width: 720px;
+
+.my-recent-wrok-bottom {
+  padding: 0 30px 0 30px;
+  display: flex;
+  flex-direction: row;
 }
 
+.mrw-img4b-desktop {
+  width: 50%;
+}
 
+.mrw-img4b-desktop img {
+  width: 100%;
+}
 
-.my-recent-wrok-buttom {
-  display: flex;
+.mrw-right-side {
+  width: 50%;
+  padding-left: 30px;
+  padding-top: 30px;
+  height: fit-content;
 }
-.img2-desktop {
-  display: block;
-  padding-left: 130px;
+
+.multi-post-stories {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 48px;
+  padding-top: 24px;
 }
-.img1-mobile {
-  display: none;
+
+.my-recent-work-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 28px;
+  color: #091e42;
+  padding-bottom: 10px;
 }
 
 /* Card  Starts Here */
 .card-master {
   display: grid;
   grid-template-columns: auto auto auto;
   margin-top: 24px;
-  padding-left: 80px; 
+  padding-left: 30px;
+}
 
+.cards-details {
+  justify-self: center;
 }
 
-.aboutme {
-  background-image: url(../images/bg-about-me-desktop.png);
-  background-position: center right ;
-  background-size: 35%;
-  background-color: #262626;;
+@media screen and (max-width: 1015px) {
```

Comment:
- [ ] kindly consider using min-width instead of max-width so as to include screens more than or equal to 768px as required below 👇 
> Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).